src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -287,7 +287,9 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
   }
 
   private _updateSetTemp(value: string): void {
-    this.shadowRoot!.querySelector("#set-temperature")!.innerHTML = value;
+    if (value !== "null") {
```

Comment:
It's not the string `"null"`, but the value `null`.

Also, we should still set a value, so just set it to empty.

Reply:
It forces it to a string, doesn't it? https://github.com/home-assistant/home-assistant-polymer/blob/0e47dace2204f5a43c24470655b9534550dc263e/src/panels/lovelace/cards/hui-thermostat-card.ts#L283

Reply:
oh you're right, that's kinda gross that it happens there and not inside this method.

Reply:
agreed, but uiValue is of type string, which is probably the reason for it. Do you want me to refactor it?

Reply:
Yeah, have it pass `null` without converting. In this method, if it's `null`, set the content to an empty string.